README.md

```diff
@@ -23,4 +23,72 @@ This repository contains various specification related to the Ethereum 1.0 chain
 | Frontier Thawing | 200000 | 09/07/2015 | | | [Blog](https://blog.ethereum.org/2015/08/04/the-thawing-frontier/) |
 | Frontier | 1 | 07/30/2015 | | | [Blog](https://blog.ethereum.org/2015/07/22/frontier-is-coming-what-to-expect-and-how-to-prepare/) |
 
+## Consensus Specification (work-in-progress)
 
+ * [Rendered specification](https://quilt.github.io/eth1.0-specs/)
+
+## Usage
+
+The Ethereum specification is maintained as a Python library, for better integration with tooling and testing.
+
+Requires Python 3.7+
+
+### Building
+
+Building the documentation is most easily done through [`tox`](https://tox.readthedocs.io/en/latest/):
+
+```bash
+$ tox -e doc
+```
+
+The path to the generated HTML will be printed to the console.
+
+### Development
+
+Running the tests necessary to merge into the repository requires:
+
+ * Python 3.7.x (not 3.8 or later), and
+ * [PyPy 7.3.x](https://www.pypy.org/).
```

Comment:
Supporting pypy is noble but I would suggest a level of pragmatism if you run into things that this makes complex.  It is probably *ok* for this spec to have a very specific version and implementation of python for which it is considered "correct".

Reply:
I think, as a mostly pure python project, it should be pretty easy to maintain compatibility, and I'd like to at least attempt to until we hit something that makes it difficult.

Ideally, we want to this project to sync mainnet, and PyPy might help speed that up. If that fails miserably, we can maybe fall back on Cython.

Reply:
If the Python code is fully typed and basically only uses the standard library, then you could also compile it to C with [mypyc](https://github.com/mypyc/mypyc) for speed. It might not be as fast as pypy (though I haven't seen any benchmarks comparing them), but mypyc seems much more actively developed than pypy.